devices/uhome.js

```diff
@@ -0,0 +1,25 @@
+const fz = require('../converters/fromZigbee');
+const tz = require('../converters/toZigbee');
+const exposes = require('../lib/exposes');
+const reporting = require('../lib/reporting');
+const ota = require('../lib/ota');
+const e = exposes.presets;
+
+module.exports = [
+    {
+        zigbeeModel: ['TWV'],
+        model: 'TWV',
+        vendor: 'UHome',
+        description: 'UHome Smart Valve',
+        ota: ota.zigbeeOTA,
```

Comment:
Do we have an OTA available for this device?

Reply:
Yes OTA cluster working. 
I've missed something with OTA?

Reply:
> Can you also provide the data/database.db entry of this device?

Sure, where can I find example of this entry?

Reply:
But we also need to have the OTA file itself this need to be added to https://github.com/koenkk/zigbee-ota

Reply:
Of course I can do that. But this is a separate repository, I wanted to do it after this was done.
Sorry for my poor git repo knowledge, do I need to pull request the OTA repository with a separate pull request or can I do it in this pull request? Same with data/database.db.
Also there is no database.db in Koenkk/zigbee2mqtt/data, only configuration.yaml.

Reply:
Can you make the pull request for the OTA first?

Reply:
Sorry for the late reply, there was an blackout. I'll try to do it this weekend if I have the light.

Reply:
The valve currently does not have a "Manufacturer ID". I am currently using the test ID "123".
Is it possible to upload such an OTA image (with manufacturer ID "123") to the repository?
If not, can we pause the OTA image upload for now and keep the OTA features?
Or just remove it from configuration file?

Reply:
Can you send me the OTA file?

Reply:
I made pull request with TWV image in OTA repository so you can see how it looks.